Client/Client.csproj

```diff
@@ -125,16 +126,13 @@ This release is based on the Zeebe 8.1.X release (https://github.com/zeebe-io/ze
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Protobuf" version="3.23.3" />
-    <PackageReference Include="Grpc" version="2.46.6" />
-    <PackageReference Include="Grpc.Auth" Version="2.54.0" />
-    <PackageReference Include="Grpc.Core" version="2.46.6" />
+    <PackageReference Include="Grpc.Net.Client" version="2.54.0" />
+    <PackageReference Include="Grpc.AspNetCore" version="2.54.0" />
+    <PackageReference Include="Grpc.Auth" version="2.54.0" />
     <PackageReference Include="Grpc.Tools" version="2.55.1">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
```

Comment:
:question: There have been some dependencies removed. How does this work now?

Reply:
The restoration process is able to manage transitive dependencies so that you don't need to specify them. If you do you will have to manage them, and ensure they are consistent with your primary dependencies.